dbos/_admin_server.py

```diff
@@ -132,7 +133,20 @@ def do_POST(self) -> None:
 
             if restart_match:
                 workflow_id = restart_match.group("workflow_id")
-                self._handle_restart(workflow_id)
+                try:
```

Comment:
Our long-term plan is to deprecate restart entirely and move to fork. I think the best change is to add a new "fork" endpoint that takes in `start_step` and then make the `restart` endpoint just call fork(1).